docs/en/guides/sre/keeper/index.md

```diff
@@ -33,57 +33,54 @@ External integrations are not supported.
-- `heart_beat_interval_ms` — How often a ClickHouse Keeper leader will send heartbeats to followers (ms) (default: 500).
-- `election_timeout_lower_bound_ms` — If the follower does not receive a heartbeat from the leader in this interval, then it can initiate leader election (default: 1000). Must be less than or equal to `election_timeout_upper_bound_ms`. Ideally they shouldn't be equal.
-- `election_timeout_upper_bound_ms` — If the follower does not receive a heartbeat from the leader in this interval, then it must initiate leader election (default: 2000).
-- `rotate_log_storage_interval` — How many log records to store in a single file (default: 100000).
-- `reserved_log_items` — How many coordination log records to store before compaction (default: 100000).
-- `snapshot_distance` — How often ClickHouse Keeper will create new snapshots (in the number of records in logs) (default: 100000).
-- `snapshots_to_keep` — How many snapshots to keep (default: 3).
-- `stale_log_gap` — Threshold when leader considers follower as stale and sends the snapshot to it instead of logs (default: 10000).
-- `fresh_log_gap` — When node became fresh (default: 200).
-- `max_requests_batch_size` - Max size of batch in requests count before it will be sent to RAFT (default: 100).
-- `max_requests_batch_bytes_size` - Max size in bytes of batch of requests that can be sent to RAFT (default: 100KiB)
-- `max_requests_append_size` - Max size of batch of requests that can be sent to replica in append request (default: 100)
-- `max_flush_batch_size` - Max size of batch of requests that can be flushed together (default: 1000)
-- `max_requests_quick_batch_size` - Max size of batch of requests to try to get before proceeding with RAFT. Keeper will not wait for requests but take only requests that are already in queue (default: 100)
-- `force_sync` — Call `fsync` on each write to coordination log (default: true).
-- `quorum_reads` — Execute read requests as writes through whole RAFT consensus with similar speed (default: false).
-- `raft_logs_level` — Text logging level about coordination (trace, debug, and so on) (default: system default).
-- `auto_forwarding` — Allow to forward write requests from followers to the leader (default: true).
-- `shutdown_timeout` — Wait to finish internal connections and shutdown (ms) (default: 5000).
-- `startup_timeout` — If the server doesn't connect to other quorum participants in the specified timeout it will terminate (ms) (default: 30000).
-- `four_letter_word_white_list` — White list of 4lw commands (default: `conf,cons,crst,envi,ruok,srst,srvr,stat,wchs,dirs,mntr,isro,rcvr,apiv,csnp,lgif,rqld,ydld`).
-- `async_replication` - Enable async replication. All write and read guarantees are preserved while better performance is achieved. Settings is disabled by default to not break backwards compatibility (default: false)
```

Comment:
It looks like there's 5 entries below `four_letter_word_white_list` that are no longer present in the new table?
Do we still need these entries?